From: "Luis Claudio R. Goncalves" <lgoncalv@redhat.com>
Date: Sat, 11 Sep 2021 12:28:50 -0300
Subject: [PATCH 347/347] Linux 4.9.282-rt187 REBASE
Origin: https://git.kernel.org/cgit/linux/kernel/git/rt/linux-stable-rt.git/commit?id=c095d5271c8b641dddb6dca24944e1c9232d69e0

Signed-off-by: Luis Claudio R. Goncalves <lgoncalv@redhat.com>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 9a218ca23053..b60208cbbe15 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt23
+-rt187
